lldb/include/lldb/Core/Progress.h

```diff
@@ -134,6 +144,9 @@ class Progress {
   /// Data needed by the debugger to broadcast a progress event.
   const ProgressData m_progress_data;
 
+  /// The origin of this progress event.
+  const Origin m_origin;
```

Comment:
This seems redundant if we store the origin in the `ProgressData`?

Reply:
Good catch! Fixed.